cypress.json

```diff
@@ -17,5 +17,6 @@
   "watchForFileChanges":false,
   "projectId": "ebzwp1",
   "pageLoadTimeout": 200000,
-  "defaultCommandTimeout": 30000
+  "defaultCommandTimeout": 30000,
+  "experimentalStudio" : true
```

Comment:
I guess we can leave this in, so long as CI still works well. @the-greenman

Reply:
Yeah. My thoughts. Can't ignore the file.